optimism.tokenlist.json

```diff
@@ -208,7 +208,7 @@
       "decimals": 8,
       "logoURI": "https://ethereum-optimism.github.io/data/APT/logo.svg",
       "extensions": {
-        "optimismBridgeAddress": "0x636Af16bf2f682dD3109e60102b8E1A089FedAa8",
+        "baseBridgeAddress": "0xfa6d8ee5be770f84fc001d098c4bd604fe01284a",
```

Comment:
As far as I know base isn't using this list yet.  I know their bridge wasn't last I checked.   Are we sure this isn't a breaking change for them?

Reply:
Reached out to someone at Base to get confirmation on this

Reply:
Confirmed with Base that this is ok to be merged.